### ISSUE TITLE
Suggesting participants for the annotation group

### ISSUE THREAD

**Issue by proycon** [CLARIAH/IG-Annotation#9]
Perhaps we can use this issue to invite/suggest participants for this group? People whose work may contribute to the discussions we have. The people mentioned can then indicate whether they are interested or not? (github will automatically poke them with a mail notification if we mention github usernames, efficient, but sorry if it's a bit too impersonal!  )

- Dirk Roorda (@dirkroorda) was mentioned before already, considering his work on TextFabric.
- It would also be good to have some representation from the VU and/or WP6, perhaps Antske Fokkens (@antske) is interested or knows somebody who is?

**Comment by dirkroorda** [comment]
Happy to get involved! Thanks for asking!

**Comment by proycon** [comment]
Great, welcome! :)

**Comment by marijnkoolen** [comment]
Yes, we need representatives from WP6 as well, e.g. @antske, @pboot and @lodewijk81. I'll ask them.
